lib/ruby_indexer/lib/ruby_indexer/configuration.rb

```diff
@@ -52,7 +59,10 @@ def indexables
       # having duplicates if BUNDLE_PATH is set to a folder inside the project structure
 
       # Add user specified patterns
-      indexables = @included_patterns.flat_map do |pattern|
+      patterns = indexable_included_file_patterns(Dir.pwd, merge_exclude_patterns).map! { |dir| File.join(dir, "*.rb") }
+      patterns = [File.join(Dir.pwd, "**/*.rb")] if patterns.empty?
```

Comment:
I'm realizing I've totally forgotten about `@included_patterns` and just treated it as hardcoded.